.gitignore

```diff
@@ -17,3 +17,4 @@ Package.resolved
 *.phpunit.cache
 *.phpunit.result.cache
 super-linter.log
+dotnetv3/SES/Tests/testsettings.local.json
```

Comment:
This should go in dotnetv3/.gitignore instead of the top-level file.
Also, wouldn't just `testsettings.local.json` be enough? Then you'll catch future files like this, too.